README.md

```diff
@@ -24,68 +27,97 @@ We like to think of it as `kubectl` for clusters.
   </a>
 </p>
 
-## Launching a Kubernetes cluster hosted on AWS
+
+## Launching a Kubernetes cluster hosted on AWS or GCE
 
 To replicate the above demo, check out our [tutorial](/docs/aws.md) for
 launching a Kubernetes cluster hosted on AWS.
 
+To install a Kuberentes cluster on GCE please follow this [guide](/docs/tutorial/gce.md).
+
 
 ## Features
 
-* Automates the provisioning of Kubernetes clusters in ([AWS](/docs/aws.md))
+* Automates the provisioning of Kubernetes clusters in [AWS](/docs/aws.md) and [GCE](/docs/tutorial/gce.md)
 * Deploys Highly Available (HA) Kubernetes Masters
-* Supports upgrading from [kube-up](/docs/upgrade_from_kubeup.md)
 * Built on a state-sync model for **dry-runs** and automatic **idempotency**
-* Ability to generate configuration files for AWS [CloudFormation](https://aws.amazon.com/cloudformation/) and Terraform [Terraform configuration](/docs/terraform.md)
+* Ability to generate [Terraform](/docs/terraform.md)
 * Supports custom Kubernetes [add-ons](/docs/addons.md)
 * Command line [autocompletion](/docs/cli/kops_completion.md)
-* Manifest Based API [Configuration](/docs/manifests_and_customizing_via_api.md)
-* Community supported!
+* YAML Manifest Based API [Configuration](/docs/manifests_and_customizing_via_api.md)
+* [Templating](/docs/cluster_template.md) and dry-run modes for creating
+ Manifests
+* Choose from eight different CNI [Networking](/docs/networking) providers out-of-the-box
+* Supports upgrading from [kube-up](/docs/upgrade_from_kubeup.md)
+* Capability to add containers, as hooks, and files to nodes via cluster manifest
 
 
-## Documentations
+## Documentation
 
 Documentation is in the `/docs` directory, [and the index is here.](docs/README.md)
 
 
+## Kubernetes Release Compatibility
+
+`kops` is intended to be backward compatible.  It is always recommended to use the 
+latest version of kop with whatever version of Kuberentes you are using. This 
+project does not follow the Kubernetes release schedule but typically releases 
+after Kubernetes does.  `kops` supports the matching Kubernetes minor release.  
```

Comment:
I would split out the "This project does not follow the kubernetes release schedule..." sentence into is own paragraph, after this paragraph.

I would also say _why_ we don't follow the kubernetes release schedule.  Something like:

"kops aims to provide a reliable installation experience for kubernetes, and typically releases about a month after the corresponding Kubernetes release.  This time allows for release issues in kubernetes, and ensures that we can support the latest features.  Kops will release alpha and beta pre-releases before then for people that are eager to try the latest kubernetes release and have environments where they can tolerate the quirks of new releases - and please do report any issues encountered."